dnf-plugins-core.spec

```diff
@@ -9,7 +10,8 @@
 %endif
 
 Name:           dnf-plugins-core
-Version:        1.0.0
+Epoch:          1
```

Comment:
moreover! we don't have epoch in dnf-plugins-extras, so thi is totally useless and jsut creating problems